### PR TITLE
Upgrade to Laravel 10

### PR DESCRIPTION
**Pull request by ahaneef29**
This PR upgraded Laravel framework to 10 version and also other supported packages

### CODE REVIEW

File: config/lighthouse.php [hunk 1]
```diff
@@ -238,7 +241,7 @@
         \Nuwave\Lighthouse\Execution\AuthenticationErrorHandler::class,
         \Nuwave\Lighthouse\Execution\AuthorizationErrorHandler::class,
         \Nuwave\Lighthouse\Execution\ValidationErrorHandler::class,
-        \Nuwave\Lighthouse\Execution\ExtensionErrorHandler::class,
+//        \Nuwave\Lighthouse\Execution\ExtensionErrorHandler::class,
```
Comment (saleem-hadad) [inline config/lighthouse.php:244]: why?

File: app/Console/Kernel.php [hunk 2]
```diff
@@ -16,6 +16,9 @@ class Kernel extends ConsoleKernel
     protected function schedule(Schedule $schedule)
     {
         $schedule->command('finance:report')->onOneServer()->lastDayOfMonth('23:00');
+
+        $schedule->command('cache:prune-stale-tags')->hourly();
```
Comment (saleem-hadad) [inline app/Console/Kernel.php:20]: what's the purpose of this command?

File: app/Http/Kernel.php [hunk 3]
```diff
@@ -16,7 +16,6 @@ class Kernel extends HttpKernel
     protected $middleware = [
         // \App\Http\Middleware\TrustHosts::class,
         \App\Http\Middleware\TrustProxies::class,
-        \Fruitcake\Cors\HandleCors::class,
```
Comment (saleem-hadad) [inline app/Http/Kernel.php:19]: removed in v10?

**Comment by saleem-hadad** [comment]
thanks, man, can you please find out why the tests are failing?

File: app/Providers/AuthServiceProvider.php [hunk 4]
```diff
@@ -23,7 +23,6 @@ class AuthServiceProvider extends ServiceProvider
      */
     public function boot()
     {
-        $this->registerPolicies();
```
[the remark this answers is not in the document]
Comment (ahaneef29) [reply app/Providers/AuthServiceProvider.php:26]: Removed based on Laravel upgrade instruction https://laravel.com/docs/10.x/upgrade#register-policies

File: app/Http/Kernel.php (hunk 3 above)
Comment (ahaneef29) [reply app/Http/Kernel.php:19]: I forgot to replace with Laravel default CORS middle and I will include in next commit.

**Comment by ahaneef29** [comment]
@saleem-hadad  My Apologies for not visited my PR. Due to lack of time. 
Thanks for merging.

**Comment by saleem-hadad** [comment]
most welcome, thanks for your time man

**Pull request merged**